core/src/client/client.rs

```diff
@@ -76,7 +76,7 @@ pub struct Client {
 
     importer: Importer,
 
-    /// Timer for reseal_min_period/reseal_max_period on miner client
```

Comment:
AFAIR, The tendermint consensus also uses reseal_max_period to prepare the situation there is no transactions in the block.
@majecty Would you check it?

Reply:
Tendermint consensus always creates a block even though there is no transaction. So it is safe to remove it.

FYI, there is a separate timer in Tendermint code to sleep a few seconds when creating an empty block. However, the timer will be removed after the Tenderand is merged.